### DISCUSSION TITLE
Add Issue Category: Asset Packs

### DISCUSSION THREAD

**Discussion by P-Star7** [Wohlstand/TheXTech#307] [Planning]
This is to categorize issues that are of relevance to creators of Asset Packs. This includes things like:

-Adding block, BGO, and effect .txts #293 
-Non-hardcoded music length #305 
-Enhancement of NPC animation #283 

that give asset creators greater latitude in customizing the game's graphics, sound effects, and music away from the default Super Mario All-Stars ones. Allowing npc-x.txt and UI graphics in asset packs are already great examples of more customization than SMBX 1.3, so I'd definitely like to see more of it.

Maybe the category should be entitled "Customization" because it is also useful for individual episodes.

**Comment by 0lhi** [comment]
Good idea. Named it "Asset Customization" (since there is no other label that starts with *A* yet) and added some Issues to it.

If I missed any, let me know.
